validate.sh

```diff
@@ -18,6 +18,11 @@ dydxprotocold_path=$(tar -xvf $tar_path --directory /tmp/dydxprotocold)
 mkdir -p ~/bin
 cp /tmp/dydxprotocold/$dydxprotocold_path /usr/local/bin/dydxprotocold
 
+NINE_ZEROS="000000000"
+EIGHTEEN_ZEROS="$NINE_ZEROS"
```

Comment:
i think there should be another nine zeros

```suggestion
EIGHTEEN_ZEROS="$NINE_ZEROS$NINE_ZEROS"
```

Reply:
Done!